src/docs/release-notes/R21/R21.mdx

```diff
@@ -0,0 +1,73 @@
+
+### Component Libraries
+
+-   Updated Angular library to build with Angular 12.
+-   Documented all the components with JSDocs, viewable with Intellisense.
+-   Added new icon library for React Native projects and enhanced / simplified [icon usage options](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--icon-wrapper) within the React Native component library.
+-   All PX Blue React Native components now responds to device rotation properly.
+-   [Angular](https://pxblue-components.github.io/angular/?path=/info/components-channel-value--readme) and [React Native](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--channel-value)'s Channel Value component now has the option to show / hide the space between unit and value.
+
+### Component Sticker Sheet
+
+-   Updated Figma [Component Sticker Sheet](https://www.figma.com/community/file/1024360297793425107) to include 60+ new device- and channel-value-related icons. You can also find all of them in our [iconography](/style/iconography) page.
+-   All icons (Material Design and PX Blue icons) are now searchable by their aliases in Figma.
+-   Added more system components, including: phone keyboard, phone keypad, status bar, browser window, cursor, system dialog box and time picker.
+-   Added table components.
+
+### Themes
+
+-   Added a new `stroked` Angular Material chip style variant.
+-   Enhanced showcase demos for [Angular](https://github.com/pxblue/angular-showcase-demo/tree/master) and [React](https://github.com/pxblue/react-showcase-demo) to display all PX Blue and Material components in an organized manner.
+-   Simplified [Dark Theming for React Native](https://github.com/pxblue/react-native-themes/tree/master#dark-theme) (eliminated the need for alternate dark theme and provide special pre-themed wrappers for React Native Paper components).
+
+### Documentation Site
+
+-   Updated Angular and React design pattern code examples for: [Account Menu](/patterns/account-menu), [App Bar](/patterns/appbar), [Forms & Validations](/patterns/forms) and [Loading & Waiting](/patterns/loading).
+
+### Re-usable Authentication Workflows
+
+-   Angular, React and React Native authentication workflows' translations are now synced up.
+-   In all three authentication workflows, users can now go back and adjust their previous inputs during registration.
+-   Reordered screens in the self-registration process so accepting the EULA appears as the first step.
+
+### PX Blue CLI
+
+-   Updated to use the latest dependencies.
+
+### Miscellaneous
+
+-   Bug fixes & minor updates.
+-   Added copyright information to all our package source code.
+-   Reorganized [icons](https://github.com/pxblue/icons) and [themes](https://github.com/pxblue/themes) repository.
+-   Improved code coverage with unit testing and Cypress.
+
+### Package Summary
```

Comment:
I like adding this section. Would it be more readable if we made the lists into a table for each framework (Package...Versions Published...Changelog)?

Reply:
Attempting to use tables in Docit directly results in this

Reply:
![image](https://user-images.githubusercontent.com/8997218/135332127-1e14d348-d2d6-47e5-bc04-528ffa32174a.png)

Reply:
It would take a few moment to stylize it like Material Design, and make it mobile friendly...

I would drop it... for this release

Reply:
yeah that's fine...you could also make a quick MDX component (using the MUI table) and pass a JSON object to populate it. No CSS required.